app/uk/gov/hmrc/helptosavefrontend/controllers/RegisterController.scala

```diff
@@ -120,6 +133,16 @@ class RegisterController @Inject() (val messagesApi:           MessagesApi,
     }
   }(redirectOnLoginURL = FrontendAppConfig.checkEligibilityUrl)
 
+  private def checkIfAccountCreateAllowed(ifAllowed: ⇒ Future[Result], ifNotAllowed: ⇒ Future[Result])(implicit hc: HeaderCarrier) = {
```

Comment:
do we need a `ifNotAllowed` function to be passed in here. Won't it always be:
```scala
SeeOther(routes.RegisterController.getUserCapReachedPage().url)
```

Reply:
fixed